src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -463,6 +470,110 @@ PRIMARY KEY ({primaryKeys})
             }
         }
 
+        async Task<ScaleMetrics> IScaleMonitor.GetMetricsAsync()
+        {
+            return await this.GetMetricsAsync();
+        }
+
+        public async Task<SqlTriggerMetrics> GetMetricsAsync()
```

Comment:
How often are metrics collected anyways? Is that a static value?

Reply:
In the video recordings explaining the scale controller, it was mentioned that the polling happens every 10 seconds. I will try to check the code and paste the link to it here but you may not have access to the repo.

Reply:
From code walkthrough too, I found it to be 10 seconds. Links to code:
- [RuntimeScaleActivationTask.cs](https://dev.azure.com/msazure/One/_git/AAPT-Antares-Websites?path=/src/Hosting/Azure/ScaleController/Microsoft.Azure.AppService.ScaleController.Extension/ActivationTasks/RuntimeScaleActivationTask.cs&version=GBdev&line=45&lineEnd=49&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)
- [ScaleControllerConfiguration.cs](https://dev.azure.com/msazure/One/_git/AAPT-Antares-Websites?path=/src/Hosting/Azure/ScaleController/Microsoft.Azure.AppService.ScaleController.Extension/Shared/ScaleControllerConfiguration.cs&version=GBdev&line=96&lineEnd=108&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)